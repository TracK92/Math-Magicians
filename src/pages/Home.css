* {
  font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
}

h2 {
  padding-bottom: 30px;
}

p {
  padding: 20px 30px;
}

.container {
  padding: 20px;
  display: flex;
  justify-content: space-between;
}

.container img {
  height: 300px;
}

@media screen and (max-width: 750px) {
  .container {
    flex-direction: column;
  }
}
