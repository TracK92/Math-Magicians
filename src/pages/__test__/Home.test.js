import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import Home from '../Home';

test('should render the home page', () => {
  const tree = render(<Router><Home /></Router>);
  expect(tree).toMatchSnapshot();
});
