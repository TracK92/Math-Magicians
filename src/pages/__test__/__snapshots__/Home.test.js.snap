// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the home page 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="homeContainer"
      >
        <h2>
          Welcome to our page!
        </h2>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Obcaecati nam quis expedita, reiciendis quod hic nobis ipsam dicta sapiente rem error temporibus molestiae itaque qui iste quasi quidem? Unde, debitis sit? Perferendis, fugiat dicta necessitatibus distinctio placeat asperiores dolore corrupti dignissimos neque! Aperiam magni iusto hic voluptatibus quos. Ut, laborum? Nesciunt aspernatur quibusdam saepe cupiditate porro impedit. Necessitatibus rem similique asperiores, nemo illum voluptatibus, ipsa aspernatur saepe repudiandae molestiae dicta! Tenetur deleniti a sunt in culpa eum illum aut adipisci?
        </p>
        <p>
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Enim quaerat omnis quo aspernatur sequi atque molestias assumenda, doloribus eum rem esse alias aperiam aliquam sapiente voluptatum sit quasi corporis provident quisquam illum. Atque dolore magni exercitationem facilis quod voluptas assumenda nostrum sint, explicabo incidunt! Dolorum ullam pariatur optio natus asperiores, aspernatur nisi, a atque doloribus vitae dicta unde nihil voluptates officiis in veritatis tenetur rerum reprehenderit maxime ea saepe modi enim magnam cupiditate. Quasi iusto in praesentium ullam esse et!
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="homeContainer"
    >
      <h2>
        Welcome to our page!
      </h2>
      <p>
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Obcaecati nam quis expedita, reiciendis quod hic nobis ipsam dicta sapiente rem error temporibus molestiae itaque qui iste quasi quidem? Unde, debitis sit? Perferendis, fugiat dicta necessitatibus distinctio placeat asperiores dolore corrupti dignissimos neque! Aperiam magni iusto hic voluptatibus quos. Ut, laborum? Nesciunt aspernatur quibusdam saepe cupiditate porro impedit. Necessitatibus rem similique asperiores, nemo illum voluptatibus, ipsa aspernatur saepe repudiandae molestiae dicta! Tenetur deleniti a sunt in culpa eum illum aut adipisci?
      </p>
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Enim quaerat omnis quo aspernatur sequi atque molestias assumenda, doloribus eum rem esse alias aperiam aliquam sapiente voluptatum sit quasi corporis provident quisquam illum. Atque dolore magni exercitationem facilis quod voluptas assumenda nostrum sint, explicabo incidunt! Dolorum ullam pariatur optio natus asperiores, aspernatur nisi, a atque doloribus vitae dicta unde nihil voluptates officiis in veritatis tenetur rerum reprehenderit maxime ea saepe modi enim magnam cupiditate. Quasi iusto in praesentium ullam esse et!
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
